src/pages/home/categories/categories.js

```diff
@@ -14,14 +14,19 @@ const Categories = () => {
   }));
   const styles = useStyles();
 
-  const items = categories.map(({ _id, name, images, categoryCode }) => (
-    <CategoryItem
-      key={_id}
-      categoryCode={categoryCode}
-      categoryName={name[LANGUAGE].value}
-      categoryImage={images.large}
-    />
-  ));
+  const CategoriesView = () => {
```

Comment:
Please, remove this component, it is really very useless one

Reply:
Fixed